### DISCUSSION TITLE
Split out retention implementations in separate modules

### DISCUSSION THREAD

**Discussion by WLefever-Cegeka** [Informatievlaanderen/VSDS-LDESServer4J#695] [Ideas]
Split out retention implementations in separate modules so that users can define their own retention policies.
See this comment: https://github.com/Informatievlaanderen/VSDS-LDESServer4J/pull/613#discussion_r1193682478

**Comment by Yalz** [comment]
Could you elaborate why? This is core domain functionality. I don't see why you would add even more modules to it
